source/simulator/initial_conditions.cc

```diff
@@ -214,6 +91,55 @@ namespace aspect
 
         std::vector<types::global_dof_index> local_dof_indices (finite_element.dofs_per_cell);
 
+#if DEAL_II_VERSION_GTE(8,5,0)
```

Comment:
The tester currently only tests deal.II 8.4.0. Did you test that this changed code works with 8.5.0.pre? If you do not have a development version of deal.II I could also do that for you. Let me know.